src/libsystemd/sd-netlink/netlink-message.c

```diff
@@ -739,7 +798,7 @@ int sd_netlink_message_enter_container(sd_netlink_message *m, unsigned short typ
         } else
                 return -EINVAL;
 
-        r = rtnl_message_read_internal(m, type, &container);
+        r = rtnl_message_read_internal(m, type, &container, &flags);
```

Comment:
Instead of catching `flags` into a throw-away variable, you could just use `NULL` here, right?